synapse/federation/federation_client.py

```diff
@@ -810,7 +810,7 @@ async def _do_send_invite(
                         "User's homeserver does not support this room version",
                         Codes.UNSUPPORTED_ROOM_VERSION,
                     )
-            elif e.code == 403:
+            elif e.code in (403, 429):
                 raise e.to_synapse_error()
```

Comment:
Cool, thanks for added. This won't pass through `retry_after_ms` but I presume the client will have some default amount of time they'll wait for.

Reply:
Oh, hmm, why on earth don't we proxy through all the fields??

I don't think clients actually care about `retry_after_ms`...